ChainReactionBindings/Bindings/TopMenuBindings/TopMenuBindings.cs

```diff
@@ -64,6 +57,7 @@ public void SeeBasketPopupIsOpen()
 		public void SeeMessageInBasketPopup(string message)
 		{
 			Assert.AreEqual(message, page.BasketMessageElement.Text);
+			Thread.Sleep(TimeSpan.FromSeconds(5));

```

Comment:
Consider using explicit feature file step for waiting, it's not good to hide this sleep logic